### PR TITLE
[Test_0]: Add Hello World Test Pr

### PR DESCRIPTION
**Pull request by JJackMoO**
### What?

- brief: Add Hello World Pr
- author: JackMo

### What changes are included in this PR?

Add Hello World in main.cpp

### Are these changes tested?

image

### CODE REVIEW

**Comment by Light-City** [comment]
pr title is wrong.

**Comment by Light-City** [comment]
don't change my pr and use my pr.

File: stl/test_team/jackmo/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+int main()
+{
```
Comment (martinx) [inline stl/test_team/jackmo/src/main.cpp:4]: keep`{` in the same line

```cpp
int main() {
```

File: stl/test_team/jackmo/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    std::cout << "Hello World~" << std::endl;
```
Comment (martinx) [inline stl/test_team/jackmo/src/main.cpp:5]: Indent 2 spaces

**Comment by martinx** [comment]
LGTM

**Comment by martinx** [comment]
Sorry, the pr title is not correct, please rename

[Test_0]   =>  [team_test_0]

thx.

**Pull request closed without merging**